src/components/authors-list.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/core';
+import React from 'react';
+import Author from './author';
+
+const list = css`
+  display: flex;
+  flex-wrap: wrap;
+  margin: 5rem 0;
+  align-items: center;
+  color: var(--gray7);
+  text-transform: uppercase;
+
+  span {
+    margin: 0.5rem;
+  }
+`;
+
+type Props = {
+  authors: string[];
+};
+
+const AuthorsList = ({ authors }: Props) => {
+  if (!authors) {
+    return null;
+  }
+
+  return (
+    <div css={list}>
```

Comment:
Even though we're in JSX lets not forget about semantic best practices 🙂 Can this be a `<u>` and the authors can be `<li>`s? The `Contributors` title can go in an `h4` before it.

Reply:
Would you be OK if I handle this in a separate PR?